app/views/consent_forms/review_match.html.erb

```diff
@@ -21,7 +21,22 @@
           patient: @patient_session.patient) %>
 
     <%= render AppCardComponent.new(heading: "Consent response") do
-      render AppConsentDetailsComponent.new(consents: [@consent_form])
+      render AppConsentSummaryComponent.new(
+        name: @consent_form.parent_name,
+        relationship: @consent_form.who_responded,
+        contact: {
+          phone: @consent_form.parent_phone,
+          email: @consent_form.parent_email
+        },
+        response: {
+          text: @consent_form.summary_with_route,
+          timestamp: @consent_form.recorded_at
+        },
+        refusal_reason: {
+          reason: @consent_form.reason,
+          notes: @consent_form.reason_notes
+        }
```

Comment:
If consent is given then this `refusal_reason` section gets included, and I think the rendering isn't quite right since it'll show an empty "Refusal reason" entry on the "Link consent response ..." page. Tested locally:

<img width="632" alt="image" src="https://github.com/nhsuk/manage-childrens-vaccinations/assets/15608/ecbc6585-8e56-457e-bfe3-e0ce03519918">

Reply:
Ah yes, I came across this in the subsequent PR and fixed it in 928d015c52755018b5c3ed83cf3ad34f4542d2a8. I've cherry-picked that commit onto this branch: [5d6004d](https://github.com/nhsuk/manage-childrens-vaccinations/pull/1206/commits/5d6004dc45d2e287c31f57d02d15677dfe5caea7)